src/Travis/Client.php

```diff
@@ -59,6 +65,16 @@ public function fetchRepository($slug)
         return $repository;
     }
 
+    public function restartBuild($build) {
```

Comment:
Because there is no instance of `$client` within a `Build` there are two options:
1. Pass the client to the build (not great) `$repsotory->getLastBuild()->restart($client);`
2. Add a `__construct()` to both `Repository` and `Build` to allow `$build->restart();` (better)

Suggesting going for the full meal deal and allow for Travis objects to perform operations. The build restart action is just the first of many.  Question: it ok to go forward with this pull request before doing so?